server/graphql/v2/mutation/MemberInvitationMutations.js

```diff
@@ -0,0 +1,151 @@
+import { GraphQLBoolean, GraphQLNonNull, GraphQLString } from 'graphql';
+import { pick } from 'lodash';
+
+import MemberRoles from '../../../constants/roles';
+import models from '../../../models';
+import { Forbidden, Unauthorized } from '../../errors';
+import { MemberRole } from '../enum';
+import { AccountReferenceInput, fetchAccountWithReference } from '../input/AccountReferenceInput';
+import {
+  fetchMemberInvitationWithReference,
+  MemberInvitationReferenceInput,
+} from '../input/MemberInvitationReferenceInput';
+import { MemberInvitation } from '../object/MemberInvitation';
+import ISODateTime from '../scalar/ISODateTime';
+
+const memberInvitationMutations = {
+  inviteMember: {
+    type: new GraphQLNonNull(MemberInvitation),
+    description: 'Invite a new member to the Collective',
+    args: {
+      memberAccount: {
+        type: GraphQLNonNull(AccountReferenceInput),
+        description: 'Reference to an account for the invitee',
+      },
+      account: {
+        type: GraphQLNonNull(AccountReferenceInput),
+        description: 'Reference to an account for the inviting Collective',
+      },
+      role: {
+        type: GraphQLNonNull(MemberRole),
+        description: 'Role of the invitee',
+      },
+      description: {
+        type: GraphQLString,
+      },
+      since: {
+        type: ISODateTime,
```

Comment:
I was the one introducing the confusion with this scalar, sorry about that. We're getting rid of it soon. Let's replace it with:

> import { GraphQLDateTime } from 'graphql-iso-date';

```suggestion
        type: GraphQLDateTime,
```